---
title: Statistical methods for detecting differentially methylated regions based on
  MethylCap-seq data
date: '2015-01-01'
draft: true
publishDate: '2021-11-08T03:10:10.682776Z'
authors:
- D. N. Ayyala
- D. E. Frankhouser
- J. Ganbat
- G. Marcucci
- R. Bundschuh
- P. Yan
- S. Lin
publication_types:
- '2'
abstract: DNA methylation is a well-established epigenetic mark, whose pattern throughout
  the genome, especially in the promoter or CpG islands, may be modified in a cell
  at a disease stage. Recently developed probabilistic approaches allow distributing
  methylation signals at nucleotide resolution from MethylCap-seq data. Standard statistical
  methods for detecting differential methylation suffer from ‘curse of dimensionality’
  and sparsity in signals, resulting in high false-positive rates. Strong correlation
  of signals between CG sites also yields spurious results. In this article, we review
  applicability of high-dimensional mean vector tests for detection of differentially
  methylated regions (DMRs) and compare and contrast such tests with other methods
  for detecting DMRs. Comprehensive simulation studies are conducted to highlight
  the performance of these tests under different settings. Based on our observation,
  we make recommendations on the optimal test to use. We illustrate the superiority
  of mean vector tests in detecting cancer-related canonical gene pathways, which
  are significantly enriched for acute myeloid leukemia and ovarian cancer.
featured: false
publication: '*Briefings in Bioinformatics*'
url_pdf: https://doi.org/10.1093/bib/bbv089
doi: 10.1093/bib/bbv089
---

